bundles/org.openhab.binding.mystrom/src/main/resources/OH-INF/update/update.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="mystrom:mystromplug">
+		<instruction-set targetVersion="1">
+			<update-channel id="ws-channel">
+				<type>system:electrical-energy</type>
```

Comment:
This is not the channel type actually used:
```suggestion
				<type>mystrom:ws-channel</type>
```

Or change the `ws` channel to use channel type `system:electric-energy`.

Additionally I would really advise to test the migration to make sure the channel is added as expected.